client/checkout/platform-checkout/express-button/woopay-express-checkout-payment-method.js

```diff
@@ -34,7 +34,13 @@ const wooPayExpressCheckoutPaymentMethod = () => ( {
 			emailSelector="#email"
 		/>
 	),
-	canMakePayment: () => true,
+	canMakePayment: () => {
```

Comment:
nit: a bit more concise version
```suggestion
	canMakePayment: () => typeof wcpayConfig !== 'undefined'
```